import-export-cli/cmd/mi/secret/create.go

```diff
@@ -0,0 +1,143 @@
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package secret
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+	"syscall"
+
+	"github.com/spf13/cobra"
+	miUtils "github.com/wso2/product-apim-tooling/import-export-cli/mi/utils"
+	"github.com/wso2/product-apim-tooling/import-export-cli/utils"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+var keyStorePropertiesFile = miUtils.GetkeyStorePropertiesFilePath()
+var inputPropertiesfile string
+var encryptionAlgorithm string
+var outputType string
+
+const secretCreateCmdLiteral = "create"
+const secretCreateCmdShortDesc = "Encrypt secrets"
+
+const secretCreateCmdLongDesc = "Create secrets based on given arguments"
+
+var secretCreateCmdExamples = "To encrypt secret and get output on console\n" +
+	"  " + utils.ProjectName + " " + utils.MiCmdLiteral + " " + secretCmdLiteral + " " + secretCreateCmdLiteral + "\n" +
+	"To encrypt secret and get output as a .properties file (stored in the security folder in apictl executable directory)\n" +
+	"  " + utils.ProjectName + " " + utils.MiCmdLiteral + " " + secretCmdLiteral + " " + secretCreateCmdLiteral + " -o file\n" +
+	"To encrypt secret and get output as a .yaml file (stored in the security folder in apictl executable directory)\n" +
+	"  " + utils.ProjectName + " " + utils.MiCmdLiteral + " " + secretCmdLiteral + " " + secretCreateCmdLiteral + " -o k8\n" +
+	"To bulk encrypt secrets defined in a properties file\n" +
+	"  " + utils.ProjectName + " " + utils.MiCmdLiteral + " " + secretCmdLiteral + " " + secretCreateCmdLiteral + " -f <file_path>\n" +
+	"To bulk encrypt secrets defined in a properties file and get a .yaml file (stored in the security folder in apictl executable directory)\n" +
+	"  " + utils.ProjectName + " " + utils.MiCmdLiteral + " " + secretCmdLiteral + " " + secretCreateCmdLiteral + " -o k8 -f <file_path>"
+
+var secretCreateCmd = &cobra.Command{
+	Use:     secretCreateCmdLiteral,
+	Short:   secretCreateCmdShortDesc,
+	Long:    secretCreateCmdLongDesc,
+	Example: secretCreateCmdExamples,
+	Args:    cobra.NoArgs,
+	Run: func(cmd *cobra.Command, args []string) {
+		if !utils.IsFileExist(keyStorePropertiesFile) {
+			utils.HandleErrorAndExit("Key Store has not been initialized.\nExecute 'apictl mi secret init --help' for more information", nil)
+		}
+		err := validateFlags()
+		if err != nil {
+			utils.HandleErrorAndExit("Invalid flag", err)
+		}
+		initSecretInformation()
+	},
+}
+
+func init() {
+	SecretCmd.AddCommand(secretCreateCmd)
+	secretCreateCmd.Flags().StringVarP(&inputPropertiesfile, "from-file", "f", "", "Path to the properties file which contain secrets to be encrypted")
```

Comment:
Shall we change this like below?
Path to the properties file **that contains** secrets to be encrypted
or 
Path to the properties file **which contains** secrets to be encrypted